src/js/utils/apiClient.jsx

```diff
@@ -137,6 +137,23 @@ export const handleValidationErrors = (setState) => (error) => {
   return handleError(error);
 };
 
+export const mapToEmptyString = (values, valuesToSkip = []) => Object.keys(values)
```

Comment:
@alannadolny Will this support nested objects?

Reply:
After my current commit - yes 😀